src/library-authoring/collections/LibraryCollectionPage.tsx

```diff
@@ -200,7 +204,7 @@ const LibraryCollectionPage = () => {
       extraFilter={[`context_key = "${libraryId}"`, `collections.key = "${collectionId}"`]}
       overrideQueries={{ collections: collectionQuery }}
     >
-      <LibraryCollectionPageInner libraryId={libraryId} />
+      <LibraryCollectionPageInner libraryId={libraryId} collectionId={collectionId} />
```

Comment:
If we are not using meilisearch index to fetch collection metadata, we can merge `LibraryCollectionPage` and `LibraryCollectionPageInner` as we no longer need search context here. You can move `SearchContextProvider` inside `Container` in line 150.

Reply:
Thank you!!
Refactored here: [e795a2b](https://github.com/openedx/frontend-app-authoring/pull/1320/commits/e795a2b91fe2efc573f7a69297ee056c6d444ca5)